recipes/object-map.md

```diff
@@ -0,0 +1,73 @@
+import Example from '../components/md/Example';
+
+<Example reactnative>
+
+Defining Object
+```js
+const exampleObject = {
+    type: 'example',
+    value: 3,
+    children: null,
+    isEditable: false
+}
+```
+
+Accessing Object values
+```js
+exampleObject.type;
+exampleObject.value;
+```
+
+</Example>
+
+<Example flutter>
+
+In Dart, there is Map like JS Objects but it is a little bit different
+
+Defining Map
+```dart
+Map<String, dynamic> exampleMap = {
```

Comment:
You can just use `final`, dart is smart enough to figure out the type of this literal